bundler/lib/bundler/cli/gem.rb

```diff
@@ -231,6 +241,35 @@ def ask_and_set_test_framework
       test_framework
     end
 
+    def ask_and_set_ci
+      ci_template = options[:ci] || Bundler.settings["gem.ci"]
+
+      if ci_template.nil?
+        Bundler.ui.confirm "Do you want to set up automated testing for your gem? " \
+          "Continuous integration services make it easy to see if pull requests have passing tests " \
+          "before you merge them. Bundler supports these services:"\
```

Comment:
```suggestion
          "before you merge them. Bundler supports these services:" \
```

(Style: Uniform space-after-the-string.)